config/billing_rates.yml

```diff
@@ -372,4 +375,4 @@
 - { id: 280f4bf4-d652-465c-86c8-8b6897d2e281, resource_type: PostgresStandbyStorage,      resource_family: m8gd,                       location: us-west-2,      unit_price: 0.0000000000, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
 - { id: bc7e1096-6688-8978-88e7-c947e08e68c5, resource_type: PostgresStandbyStorage,      resource_family: m8gd,                       location: us-east-1,      unit_price: 0.0000000000, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
 - { id: 2befac30-f11a-8178-93c0-c7d4ecad8228, resource_type: PostgresStandbyStorage,      resource_family: m8gd,                       location: us-east-2,      unit_price: 0.0000000000, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
-- { id: 0981f65e-5f1d-40fe-a416-c5e3a544f8be, resource_type: PostgresStandbyStorage,      resource_family: m8gd,                       location: ap-southeast-2, unit_price: 0.0000000000, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
\ No newline at end of file
+- { id: 0981f65e-5f1d-40fe-a416-c5e3a544f8be, resource_type: PostgresStandbyStorage,      resource_family: m8gd,                       location: ap-southeast-2, unit_price: 0.0000000000, billed_by: duration, active_from: 2025-06-29T00:00:00Z }
```

Comment:
It should have a new line at the end according to our `.editorconfig` file.

<img width="4526" height="182" alt="CleanShot 2025-08-05 at 09 38 43@2x" src="https://github.com/user-attachments/assets/a9eefc2a-ee9a-46f3-9cbb-31df1b357470" />